Package.swift

```diff
@@ -30,13 +30,15 @@ let package = Package(
         ),
     ],
     dependencies: [
-        .package(url: "https://github.com/Esri/arcgis-maps-sdk-swift", .upToNextMinor(from: "200.5.0"))
+        .package(url: "https://github.com/Esri/arcgis-maps-sdk-swift", .upToNextMinor(from: "200.5.0")),
+        .package(url: "https://github.com/apple/swift-markdown.git", branch: "0.4.0")
```

Comment:
It seems the URL is auto redirecting to https://github.com/swiftlang/swift-markdown 👀

Reply:
I'll pull .git off the URL. I don't think it's needed.

Reply:
As Ting pointed out, the repo has been moved from the apple org to the swiftlang org. While GitHub helpfully redirects, we should update this URL so that a redirect isn't necessary.

Reply:
Also, from what I've seen, most of the time Apple includes `.git` in their dependency URLs. I don't know why, but that might also be to prevent a redirect. I suggest including it.